go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -149,6 +151,12 @@ func initMySQLProtocol() {
 	}
 	mysqlListener.AllowClearTextWithoutTLS = *mysqlAllowClearTextWithoutTLS
 
+	// Check for the connection threshold
+	if mysqlSlowConnectWarnThreshold.Nanoseconds() != 0 {
```

Comment:
I think without the .Nanoseconds() it should still work:
if mysqlSlowConnectWarnThreshold != 0 {